src/Umbraco.Web.UI.Client/src/views/propertyeditors/templatablegrid/template.prevalues.html

```diff
@@ -0,0 +1,10 @@
+﻿<div ng-controller="Umbraco.PropertyEditors.TemplatableGridPrevalueEditorController">
+    <div ng-show="unsupported">Template not supported in document type editor, please use the data type editor.</div>
+    <div ng-show="!unsupported">
+        <button class="btn btn-small" ng-click="reset()"
+                style="z-index: 999;">
```

Comment:
I have no clue what-so-ever where to put or if there is a class that sets z-index.